style.css

```diff
@@ -0,0 +1,341 @@
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 25px;
+}
+
+.Top a {
+  margin-left: 25px;
+  padding-left: 5px;
+  padding-top: 10px;
+}
+
+nav {
+  margin-right: 30px;
+  padding-right: 5px;
+  padding-top: 10px;
+}
+
+section {
+  display: grid;
+  width: 100%;
+  align-items: center;
+  margin-right: 25px;
+  margin-left: 25px;
+  padding: 20px 30px 30px 10px;
+  grid-gap: 50px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  padding: 10px 20px 10px 20px;
+  font-family: "Poppins", sans-serif;
+  background-image: url("images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+.works {
+  display: flex;
+  flex-flow: column;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin-left: 10px;
+}
+
+.aboutme {
+  background-color: white;
+}
+
+.works > img {
+  order: -1;
+  align-self: center;
+  padding-top: 50px;
+}
+
+.sociol-logo {
+  display: flex;
+  flex-direction: row;
+}
+
+.sociol-logo ul {
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+}
+
+.sociol-logo ul li {
+  float: left;
+  padding-left: 20px;
+}
+
+.works-detail {
+  display: flex;
+  flex-direction: column;
+  font-family: "Poppins", sans-serif;
+  padding: 20px 30px 50px 20px;
+}
+
+.works-detail .main-topic {
+  font-size: 50px;
+  font-style: normal;
+  font-weight: bold;
+  line-height: 44px;
+}
+
+.works-intro {
+  display: flex;
+  flex-direction: rows;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 5px;
+  font-size: 10px;
+  padding-left: 0;
+  margin-left: 0;
+}
+
+.works-intro ul {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  padding-left: 0;
+  padding-right: 150px;
+}
+
+.works-intro h2 {
+  padding-left: 0;
+  margin-left: 0;
+}
+
+.works-intro ul li {
+  display: flex;
+  float: left;
+  justify-content: space-between;
+  padding-left: 1em;
+}
+
+.works-intro ul li img {
+  max-width: 100%;
+  height: auto;
+}
+
+.lang-icon ul {
+  display: flex;
+  margin-left: 0;
+  list-style: none;
+  padding-left: 0;
+}
+
+.lang-icon ul li {
+  display: flex;
+  padding-left: 1em;
+  margin-left: 0;
+}
+
+.language button {
+  display: inline-block;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  border-radius: 10px;
+  width: 150px;
+  height: 50px;
+  border: 1px solid blue;
+}
+
+.works-1,
+.works-2 {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 10px 20px 10px 20px;
+  font-family: "Poppins", sans-serif;
+}
+
+button:hover {
+  background-color: blue;
+}
+
+button:active {
+  background-color: darkblue;
+}
+
+button:disabled {
+  border: 1px solid #999;
+  background-color: #ccc;
+  color: #666;
+}
+
+.aboutme .aboutme-box button {
```

Comment:
- [ ] please instead of just telling the student he can do without that, it will be better you give him/her the reason why you think they can do without that.